lib/install/validate/config.go

```diff
@@ -524,11 +524,8 @@ func (v *Validator) isStandaloneHost() bool {
 	return false
 }
 
-// drs checks that DRS is enabled
-func (v *Validator) CheckDrs(ctx context.Context) {
-	if v.DisableDRSCheck {
-		return
-	}
+// CheckDrs will validate DRS settings
```

Comment:
We should mention here that this func modifies `conf` if DRS is disabled. This would be helpful since the func name `CheckDRS` doesn't convey that it modifies an input parameter.